docs/tests/accessibility-tests.md

```diff
@@ -34,3 +35,56 @@ in your unit tests with [jest-axe](https://github.com/nickcolley/jest-axe).
 
 If you're still curious
 [there are some great talks on accessibility for developers](https://www.youtube.com/watch?v=8E9AEZjglqI).
+
+## PR Review Checklist
+
+Pull Requests should include accessibility testing to prevent barriers from
+making it into the codebase. Sometimes issues found may be out of scope for a
+particular PR – those can be opened as separate issues to be followed up on.
+
+Test the functionality in web browsers for accessibility impact: Chrome,
+Firefox, and IE 11 were popular for Windows screen reader users in 2019
+[according to WebAIM](https://webaim.org/projects/screenreadersurvey8/#browsers).
+Safari is the most common choice for Voiceover users on the Mac, even if Chrome
+is widely used for development. Mobile browsers are worth checking, too.
```

Comment:
FYI: we don't support IE11. AFAIK the website doesn't work anymore in IE11, only Edge versions based on Chromium. 

cc @nickytonline

Reply:
> FYI: we don't support IE11. AFAIK the website doesn't work anymore in IE11, only Edge versions based on Chromium. 
> 
> 
> 
> cc @nickytonline 

Correct @rhymes.

Reply:
got it! We can remove that part.